sdk-ext-lib/src/main/java/cash/z/ecc/sdk/fixture/Zip321UriParseFixture.kt

```diff
@@ -0,0 +1,22 @@
+package cash.z.ecc.sdk.fixture
+
+import cash.z.ecc.sdk.model.WalletAddress
+import cash.z.ecc.sdk.model.Zatoshi
+import cash.z.ecc.sdk.model.ZecRequest
+import cash.z.ecc.sdk.model.ZecRequestMessage
+
+object Zip321UriParseFixture {
+    const val URI: String = "zcash:ztestsapling10yy2ex5dcqkclhc7z7yrnjq2z6feyjad56ptwlfgmy77dmaqqrl9" +
+        "gyhprdx59qgmsnyfska2kez?amount=1&memo=VGhpcyBpcyBhIHNpbXBsZSBtZW1vLg&message=Thank%20you%20" +
+        "for%20your%20purchase"
+
+    const val ADDRESS: String = WalletAddressFixture.UNIFIED_ADDRESS_STRING
+    @Suppress("MagicNumber")
+    val AMOUNT = Zatoshi(123)
+    val MESSAGE = ZecRequestMessage("Thanks for lunch")
+
+    @Suppress("UNUSED_PARAMETER")
+    suspend fun new(
+        toParse: String = URI,
+    ) = ZecRequest(WalletAddress.Unified.new(ADDRESS), AMOUNT, MESSAGE)
```

Comment:
@ccjernigan I see that we ideally want to call `ZecRequest.fromUri(toParse)` here, but if I understand it right, then there is a circular method calling between `ZecRequest.fromUri(toParse)` and `Zip321UriParseFixture.new(toParse)` then. So that's why I've implemented it this way, until a proper call to the SDK project can be made.

Reply:
Ah, you're right.  I haven't written an infinite loop in a while. I was clearly overdue. 

Maybe a small comment in the code so that someone else looking at this will know what's happening here and that we plan to change it?

Reply:
Yes, I've just added them to both `Zip321UriParseFixture` and `Zip321UriBuildFixture`.